### PR TITLE
Lesson 1 6

### PR DESCRIPTION
**Pull request by larasodjati**
Created reusable input with label component, refactor input with a label to use component composition, added auto-focus to input, added remove button to list items, and created a new state for removed to-do items.

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -17,14 +18,12 @@ function AddTodoForm({ onAddTodo }) {
...
-      <input
-        type="text"
-        id="todoTitle"
-        name="title"
-        value={todoTitle}
-        onChange={handleTitleChange}
-      ></input>
+      <InputWithLabel
+        todoTitle={todoTitle}
+        handleTitleChange={handleTitleChange}
```
Comment (Atos20) [inline src/AddTodoForm.js:23]: we are missing the prop `label`

File: src/InputWithLabel.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+function InputWithLabel(props) {
+  const inputRef = React.useRef();
+
+  React.useEffect(() => {
+    inputRef.current.focus();
+  }, []);
+
+  return (
+    <>
+      <label htmlFor="todoTitle">{props.children}</label>
```
Comment (Atos20) [inline src/InputWithLabel.js:12]: once you pass the prop label, add `{props.label}` instead the props.childre.

**Review by Atos20 — COMMENTED** [review]
Great job!!

**Comment by larasodjati** [comment]
@Atos20 Thank you for the feedback! I really appreciate it.

**Pull request merged**